protos/receipt_transaction.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+syntax = "proto3";
+
+import "organization.proto";
+
+message CreateReceipt {
+
+	// Whether it was a coupon or redemption payment
+	Organization.Receipt.PaymentType payment_type = 1;
```

Comment:
Slightly unsure of how to correctly access submessages, however, when building, the Organization prefix was necessary. Same issue for holding_transaction.